lib/fluent/plugin/in_tail.rb

```diff
@@ -28,11 +28,13 @@
   Fluent::FileWrapper = File
 end
 
-module Fluent
-  class TailInput < Input
-    include SystemConfig::Mixin
+module Fluent::Plugin
+  class TailInput < Fluent::Plugin::Input
+    include Fluent::SystemConfig::Mixin
```

Comment:
This mixin is included in `Fluent::Plugin::Base`, the base class of all plugins.